src/assets/scripts/crud_todo.js

```diff
@@ -0,0 +1,15 @@
+import Todo from './todo.js';
+
+const addTodo = (id, desc) => {
+  const todo = new Todo(id, desc);
+  return todo.add();
+};
+
+const editTodo = () => {};
+
+const removeTodo = (id) => Todo.remove(id);
+
+const updateTodo = () => {};
+
+// eslint-disable-next-line object-curly-newline
```

Comment:
- [ ] Please avoid disabling linter rules, the linters help tremendously to keep a consistent code structure and enforce some best practices over your code! Kindly consider removing this comment and fixing the error. 🤓 🔧